app/views/shared/_nav_auth.html.slim

```diff
@@ -0,0 +1,27 @@
+nav.bg-white
+  .container.px2.py3
+    .clearfix
+      .sm-col.mb1.sm-m0
+        = link_to \
+          image_tag(asset_url('logo.svg'),
+            alt: APP_NAME,
+            class: 'align-bottom',
+            width: 170), root_path, class: 'inline-block'
+        .mt-12p.h6.sm-show.caps
+          = link_to 'About', root_path, class: 'mr3 gray'
+          = link_to 'News', root_path, class: 'mr3 gray'
+          = link_to 'Contact', root_path, class: 'mr3 gray'
+      .sm-col-right.sm-right-align
+        - if signed_in?
```

Comment:
it looks like in `app/views/shared/_nav.html.slim` we are only showing this view template if a user is signed in, so can we remove the conditional from this view template? (assuming we only render this from one location, which may be wrong)